model/eligibility/Eligibility.php

```diff
@@ -137,6 +138,15 @@ public function getId()
         return $this->id;
     }
 
+    /**
+     * @return bool
+     * @throws \core_kernel_persistence_Exception
+     */
+    public function getProctored(){
+        $byPass =  $this->getResource($this->id)->getOnePropertyValue(new Property( EligibilityService::PROPERTY_BYPASSPROCTOR_URI));
```

Comment:
`common_exception_Error` is not handled or mentioned in the docblock.

Reply:
The variable name should reflect the value type. What about `isProctorBypassed`?

Reply:
Please remove the space before `EligibilityService::PROPERTY_BYPASSPROCTOR_URI`.